code/image_modifications.ipynb

```diff
@@ -0,0 +1,665 @@
+{
```

Comment:
<p>I think we can get rid of <strong>img_inverted</strong>, as we aren't using it right now.</p><p> </p><p>We could also look at some adaptive threshold methods instead of setting a global value e.g. like its done <a href="https://opencv-python-tutroals.readthedocs.io/en/latest/py_tutorials/py_imgproc/py_thresholding/py_thresholding.html" target="_blank">https://opencv-python-tutroals.readthedocs.io/en/latest/py_tutor</a> here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Another (really) minor thing, is there a reason why Binary_Image / Second_Image is uppercase? </p>

Reply:
Lets commit to snake_case. Ill have a look for Jupiter Notebook linters tomorrow

Reply:
Currently looking into thresholding further, it is my assumption that it could help us to blackout the unwanted liver fat to better differentiate between the fat tissue and the liver tissue.